README.md

```diff
@@ -1,5 +1,38 @@
-# lighthouse 
-Controller to facilitate DNS discovery in multi-cluster connected environments (proof-of-concept state)
+# Lighthouse 
+The lighthouse along with Submariner facilitate DNS discovery in multi-cluster connected environments. The solution aims to compatible with any CNI-compatible cluster network provider.
```

Comment:
Lighthouse provides DNS discovery to clusters connected by Submariner

Reply:
Done